src/app/errorReports/thanks/Thanks.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
```

Comment:
IIRC in the designs this was actually a modal on top of the LocationDetails/NewErrorReportForm modal, not its own full page.